### ISSUE TITLE
On Windows

### ISSUE THREAD

**Issue by Abelon38** [elpaso/qgis-server-landing-page-plugin#40]
Hi!

Is that possible to install the plugin in a QGIS-server on a Windows machine or it will only work on Docker?

Thanks!
Abel

**Comment by pathmapper** [comment]
In general. this functionality is now part of the QGIS source code, for further information please see
-https://docs.qgis.org/3.28/en/docs/server_manual/catalog.html
-https://github.com/elpaso/qgis-server-landing-page-plugin/issues/6#issuecomment-841156037

But you need a QGIS Server build with the flag `WITH_SERVER_LANDINGPAGE_WEBAPP` enabled.

**Comment by Abelon38** [comment]
Do you mean I have to build QGIS server from scratch and enable WITH_SERVER_LANDINGPAGE_WEBAPP ? or is that possible to enable it on an existing installation? 

Thanks for your support! :) 
BR/Abel

**Comment by pathmapper** [comment]
> Do you mean I have to build QGIS server from scratch and enable WITH_SERVER_LANDINGPAGE_WEBAPP ?

Yes, but only if your existing installation wasn't build with it. Unfortunately this seems sometimes the case, e.g. with Debian/Ubuntu builds.

Don't know about the Windows builds, maybe ask on the QGIS user mailing list -> https://lists.osgeo.org/mailman/listinfo/qgis-user

**Comment by Abelon38** [comment]
Hi Alessandro! 

I found the Landing installation under my QGIS-server installation "apps\qgis\resources\server\src" Then with npm  installed all modules and with Yarn run Build. After that I got a build under apps\qgis\resources\server\api\ogc\static\landingpage. Next step, allowing variables  like this:

FcgidInitialEnv QGIS_SERVER_LANDING_PAGE_PREFIX "landing"
 FcgidInitialEnv WITH_SERVER_LANDINGPAGE_WEBAPP true
 FcgidInitialEnv QGIS_SERVER_LANDING_PAGE_PROJECTS_DIRECTORIES "C:\qgisprojekt\bakgrunder2.qgz"

Still nothing happens. I am starting believe Landing-page will only work on linux environments. 

Catalog seams to be a great impruvement to the QGIS-server but I think more information regarding installation and configuration should be writen and shared otherwise it may be difficult for the users to test and use it. 

Is that possible for you to guide me in the installation of this plugin? Then I will write down a steps for steps for the QGIS public and share it. I could pay for one hour of your time if it's need it?

Thanks and I hope to hear from you.
email: abel.gonzalez.c@gmail.com

BR/Abel
